src/components/filters/DateFilter/DateFilterBody/tests/DateFilterBody.spec.tsx

```diff
@@ -64,4 +64,42 @@ describe("ExtendedDateFilterBody", () => {
         const rendered = shallow(<DateFilterBody {...mockProps} isEditMode={false} />);
         expect(rendered.find(EditModeMessage)).not.toExist();
     });
+
+    describe("calculateHeight", () => {
+        const { innerWidth, innerHeight } = window;
```

Comment:
should we move this to `afterAll`?

Reply:
No, we should not. 
This should be run at the begining of the tests, to cache the size of the window.